src/npc.cpp

```diff
@@ -413,8 +431,12 @@ bool Npc::getNextStep(Direction& dir, uint32_t& flags)
 	return getRandomStep(dir);
 }
 
-void Npc::setIdle(bool idle)
+void Npc::setIdle(const bool idle)
```

Comment:
I don't know if the const is necessary, but I like it anyway ;)

Reply:
It's not necessary, just good practice to use const correctness

https://lmgtfy.app/?q=const+correctness+c%2B%2B